src/main/java/com/notify/notify/service/SMSCRUDService.java

```diff
@@ -0,0 +1,60 @@
+package com.notify.notify.service;
+
+import com.notify.notify.exceptions.InValidRequestException;
+import com.notify.notify.exceptions.NotFoundException;
+import com.notify.notify.model.SMS;
+import com.notify.notify.model.request.SendSMSRequest;
+import com.notify.notify.repository.SMSRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+
+@Service
+public class SMSCRUDService {
+
+    @Autowired
+    private SMSRepository smsRepository;
+
+    public Long saveSMS(SendSMSRequest sendSmsRequest)
+    {
+        if(sendSmsRequest.getPhoneNumber()==null||sendSmsRequest.getPhoneNumber().length()!=10)
```

Comment:
to check null and length of sting you can use field levelvalidation  while receiving request 
please read about this here:
https://reflectoring.io/bean-validation-with-spring-boot/